editor.planx.uk/src/routes/team.tsx

```diff
@@ -74,6 +74,12 @@ const routes = compose(
       return import("./flow");
     }),
 
+    "/:flow/feedback": lazy(() => import("./feedback")),
+
+    "/:flow/service": lazy(() => import("./serviceSettings")),
+
+    "/:flow/submissions-log": lazy(() => import("./submissionsLog")),
+
```

Comment:
These needed to be moved as they do not require `FlowEditor` as part of their layout. They now exist within the same parent view.

Reply:
One strange thing I'm noticing here that I suspect is related to this:
- Open a team > Open a flow > Navigate to `/:flow/submissions-log` from menu side icons
- Refresh page from submissions log > now Team settings menu is visible, not the flow side icons! 

A bit disorienting / unclear how to "go back" ! 
![Screenshot from 2024-09-16 10-23-12](https://github.com/user-attachments/assets/cf51fee0-e0c9-43e0-bbc9-5f60a8c4a97f)

Reply:
`:flow/service` does the same thing, while `:flow/feedback` throws a "Not found" error page refresh (not new I think).

Reply:
Thanks - very well spotted. I can recreate this as described, taking a look.

Reply:
This should now be fixed by [4351014](https://github.com/theopensystemslab/planx-new/pull/3671/commits/4351014b22e4817e17c6c9690f44265f6d0ebb81)

`EditorNavMenu` relies on `flowSlug`, which wasn't getting set on a refresh.

A wider refactor to move these routes to `/:flow/settings/:page` might be a more robust approach, but the above solution works for me - we can always take another look at this if the number of pages grow, or offload the responsibility to the nav menu itself.